libs/sdk-ui-ext/src/internal/components/pluggableVisualizations/pivotTable/widthItemsHelpers.ts

```diff
@@ -147,6 +149,19 @@ function adaptWidthItemsToPivotTable(
                 },
             };
 
+            // need to filter this too, so it takes in consideration previous row bucket identifiers that were moved to column bucket
+            // as same as is done with referencePoint
+            const filteredColumnAttributeLocalIdentifiers: string[] = [];
```

Comment:
can this be conditioned by isAdaptPropertiesToInsight?

Reply:
Yes, updated it.